file2.txt

```diff
@@ -1 +1,2 @@
 this is a code in file2
+i am the participant and i added this ligne
```

Comment:
what is that ?

Reply:
@maronkaaam

Reply:
i have seen this due to my expertise i propose this to you @MZgeek